rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -767,6 +839,62 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_distribution_processor_with_v1_distribution_message() {
```

Comment:
Maybe makes sense to replicate the test for a set as well

Reply:
Yes, adding that